crates/volta-core/src/platform/sourced.rs

```diff
@@ -8,8 +8,14 @@ use semver::Version;
 use volta_fail::Fallible;
 
 pub enum Source {
+    /// Represents a Platform that came from the user default
```

Comment:
I'm wondering: given how our terminology has standardized, should we consider renaming `User` to `Default` throughout, since both project tools and default tools are part of the user toolchain?

Reply:
Definitely agree here! I'll update this list and also create a follow-on issue to look at cleaning that naming up throughout the code base.